usr/share/rear/prep/NETFS/GNU/Linux/205_inspect_tar_capabilities.sh

```diff
@@ -0,0 +1,37 @@
+# prep/NETFS/GNU/Linux/205_inspect_tar_capabilities.sh
+# The purpose is to inspect the 'tar --usage' for certain capabilities and if found
+# add these to the BACKUP_PROG_OPTIONS array
+# See also issue https://github.com/rear/rear/issues/1175
+
+# As we will only inspect 'tar' we make one big if block:
+
+if [[ "$(basename $BACKUP_PROG)" = "tar" ]] ; then
```

Comment:
I guess @jsmeix would suggest here to check for tar or return out of this script instead.